src/zrc/api/viewsets.py

```diff
@@ -103,7 +103,7 @@
 )
 
 
-@conditional_retrieve()
+@conditional_retrieve(extra_depends_on={"status"})
```

Comment:
Shouldn't this also include `resultaat`?

Reply:
that should be picked up automatically through the serializer relation: https://github.com/VNG-Realisatie/zaken-api/blob/master/src/zrc/api/serializers/core.py#L996 which can be resolved easily to a model field.

`extra_depends_on` is meant for things like `SerializerMethodField` that cannot be automatically introspected.